### PR TITLE
9228: BULK ORDERING - Add a Catalogue Solution - one list price

### PR DESCRIPTION
**Pull request by talawson05**
Added two new scenarios for skipping list price, updated expected service recipient steps

### CODE REVIEW

File: src/OrderFormAcceptanceTests.Steps/Steps/CatalogueSolutions.cs [hunk 1]
```diff
@@ -47,6 +47,15 @@ public void GivenThereIsNoCatalogueSolutionInTheOrder()
             searchedOrderItem.Should().BeEmpty();
         }
 
+        [Given(@"a supplier which has a catalogue soltution with only one list price was chosen")]
+        public void GivenASupplierWhichHasACatalogueSoltutionWithOnlyOneListPriceWasChosen()
+        {
+            var order = (Order)Context["CreatedOrder"];
+            order.SupplierId = 100001;
```
Comment (PhysicsJon) [inline src/OrderFormAcceptanceTests.Steps/Steps/CatalogueSolutions.cs:54]: Can this be grabbed from the db rather than strongly typed?

**Review by PhysicsJon — CHANGES_REQUESTED** [review]

File: src/OrderFormAcceptanceTests.Steps/Steps/CatalogueSolutions.cs (hunk 1 above)
Comment (talawson05) [reply src/OrderFormAcceptanceTests.Steps/Steps/CatalogueSolutions.cs:54]: which would you prefer, use the Id to get the name, or use the name to get the Id. Or are you wanting a query which asks for the all suppliers which have a catalogue solution with only one price and then takes the first one?

Comment (PhysicsJon) [reply src/OrderFormAcceptanceTests.Steps/Steps/CatalogueSolutions.cs:54]: The latter I think. Take a look at the query that Ruth added and change the Order By direction and you should get a single solution supplier

Comment (talawson05) [reply src/OrderFormAcceptanceTests.Steps/Steps/CatalogueSolutions.cs:54]: Where is that, in her current PR or already in the code somewhere?

Comment (PhysicsJon) [reply src/OrderFormAcceptanceTests.Steps/Steps/CatalogueSolutions.cs:54]: Current PR

**Review by PhysicsJon — APPROVED** [review]

**Comment by talawson05** [comment]
AB#9330

**Pull request merged**